ketcher-autotests/tests/Structure-Creating-&-Editing/SMARTS-attributes/Reaction-flags-attributes/reaction-flags-attributes.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { Page, test } from '@playwright/test';
+import {
+  BondTypeName,
+  clickInTheMiddleOfTheScreen,
+  doubleClickOnAtom,
+  pressButton,
+  selectBond,
+  takeEditorScreenshot,
+  waitForAtomPropsModal,
+  waitForPageInit,
+} from '@utils';
+import {
+  checkSmartsValue,
+  checkSmartsWarnings,
+  setReactionFlagExactChange,
+  setReactionFlagInversion,
+} from '../utils';
+
+const defaultFileFormat = 'MDL Molfile V2000';
+const expectedSmarts = '[#6](-[#6])(-[#6])-[#6]';
+
+async function drawStructure(page: Page, numberOfClicks: number) {
+  await selectBond(BondTypeName.Single, page);
+  for (let i = 0; i < numberOfClicks; i++) {
+    await clickInTheMiddleOfTheScreen(page);
+  }
+}
+
+async function drawStructureWithCustomBondNumberAndDoubleClickOnAtom(
+  page: Page,
```

Comment:
Maybe we could still use `drawStructureAndDoubleClickOnBond` since there is only one param `customBondNumber ` difference. We could check if there is `customBondNumber` param and call different `drawStructure` func.